src/main/resources/lib/ssb/utils/variantUtils.ts

```diff
@@ -139,13 +139,32 @@ function calculateQuarter(previousFrom: string, language: string): string {
   const date: Date = new Date(previousFrom)
   const fromMonth: number = date.getMonth() + 1
   const quarter: number = Math.ceil(fromMonth / 3)
+
+  if (language === 'en') {
+    const suffix = getSuffix(quarter)
+    return `figures for ${quarter}${suffix} Quarter ${date.getFullYear()}`
```

Comment:
This string will overwrite the translation in the english phrases. Meaning that if this phrase get changed in the phrases file, the change won't be reflected. Se suggestion below.

Reply:
added changes so phrases_en.properties does not get overwritten